storage/range_command.go

```diff
@@ -306,19 +306,20 @@ func (r *Range) EndTransaction(batch engine.Engine, ms *engine.MVCCStats, args p
 		return reply, err
 	}
 
+	// Resolve any explicit intents.
+	for _, intent := range args.Intents {
```

Comment:
Might want a TODO here to differentiate between intents which fall within this range and otherwise.